<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelineWidget</class>
 <widget class="QWidget" name="TimelineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timeline</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="timeSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="pageStep">
        <number>250</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="currentTimeSpinBox">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QToolButton" name="playButton">
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="../../../core/ui/Icons.qrc">
         <normaloff>:/data/resources/icons/play.png</normaloff>:/data/resources/icons/play.png</iconset>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="pauseButton">
       <property name="text">
        <string>||</string>
       </property>
       <property name="icon">
        <iconset resource="../../../core/ui/Icons.qrc">
         <normaloff>:/data/resources/icons/pause.png</normaloff>:/data/resources/icons/pause.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="stopButton">
       <property name="text">
        <string>S</string>
       </property>
       <property name="icon">
        <iconset resource="../../../core/ui/Icons.qrc">
         <normaloff>:/data/resources/icons/stop.png</normaloff>:/data/resources/icons/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Working...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../core/ui/Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>timeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TimelineWidget</receiver>
   <slot>timeSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>pressed()</signal>
   <receiver>TimelineWidget</receiver>
   <slot>playButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>pressed()</signal>
   <receiver>TimelineWidget</receiver>
   <slot>stopButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>pressed()</signal>
   <receiver>TimelineWidget</receiver>
   <slot>pauseButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>TimelineWidget</receiver>
   <slot>timeSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>TimelineWidget</receiver>
   <slot>timeSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>timeSliderChanged(int)</slot>
  <slot>playButtonPressed()</slot>
  <slot>pauseButtonPressed()</slot>
  <slot>stopButtonPressed()</slot>
  <slot>timeSliderPressed()</slot>
  <slot>timeSliderReleased()</slot>
 </slots>
</ui>
